main.js

```diff
@@ -23,12 +23,18 @@ const URL = require('url');
 const config = require('./app/features/config');
 const { openExternalLink } = require('./app/features/utils/openExternalLink');
 const pkgJson = require('./package.json');
+const OS = require('os');
 
 const showDevTools = Boolean(process.env.SHOW_DEV_TOOLS) || (process.argv.indexOf('--show-dev-tools') > -1);
 
 // We need this because of https://github.com/electron/electron/issues/18214
 app.commandLine.appendSwitch('disable-site-isolation-trials');
 
+// We need to disable hardware acceleration on Windows because its causes the screenshare to flicker.
+if (OS.platform() === 'win32') {
```

Comment:
Can you use `process.platform` instead, since it's already imported? Note how we use it to not close the last window on macOS.

Reply:
oh I didn't see that part, thanks for pointing it out.